receiver/postgresqlreceiver/integration_test.go

```diff
@@ -52,7 +52,7 @@ func integrationTest(name string, databases []string) func(*testing.T) {
 		NewFactory(),
 		scraperinttest.WithContainerRequest(
 			testcontainers.ContainerRequest{
-				Image: "postgres:9.6.24",
+				Image: "postgres:12.20",
```

Comment:
Can we support both versions?

Reply:
In test or more generally? This is only moving the minimal requirement to the last supported version but doesn't actually change anything

Reply:
I think the question is more, does this mean that backwards compatibility is preserved?

Reply:
```suggestion
				Image: "postgres:13.18",
```